packages/preview-middleware/src/base/flp.ts

```diff
@@ -639,6 +674,7 @@ export async function initAdp(
             flp.rta.layer = layer;
             flp.rta.options = {
                 projectId: variant.id,
+                telemetry: true, // temprory until passed during intiialization
```

Comment:
There should be a way for the user to disable telemetry, but now it is not possible.

Reply:
Now using central telemetrySetting